src/contractTest/java/uk/gov/hmcts/reform/wataskmanagementapi/consumer/wa/TaskManagerReconfigureTaskConsumerTest.java

```diff
@@ -0,0 +1,54 @@
+package uk.gov.hmcts.reform.wataskmanagementapi.consumer.wa;
+
+import au.com.dius.pact.consumer.MockServer;
+import au.com.dius.pact.consumer.dsl.PactDslWithProvider;
+import au.com.dius.pact.consumer.junit5.PactTestFor;
+import au.com.dius.pact.core.model.PactSpecVersion;
+import au.com.dius.pact.core.model.RequestResponsePact;
+import au.com.dius.pact.core.model.annotations.Pact;
+import io.restassured.http.ContentType;
+import net.serenitybdd.rest.SerenityRest;
+import org.junit.jupiter.api.Test;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.HttpStatus;
+import org.springframework.test.context.ContextConfiguration;
+import uk.gov.hmcts.reform.wataskmanagementapi.SpringBootContractBaseTest;
+import uk.gov.hmcts.reform.wataskmanagementapi.provider.service.CamundaConsumerApplication;
+
+@PactTestFor(providerName = "wa_task_management_api_reconfigure_task_by_id", port = "8991")
+@ContextConfiguration(classes = {CamundaConsumerApplication.class})
+public class TaskManagerReconfigureTaskConsumerTest extends SpringBootContractBaseTest {
+
+    private static final String WA_URL = "/task";
+    private static final String WA_RECONFIGURE_TASK_BY_ID = WA_URL + "/operation";
+
+    @Test
+    @PactTestFor(pactMethod = "executeReconfigureTaskById204", pactVersion = PactSpecVersion.V3)
+    void testReconfigureTaskByTaskId204Test(MockServer mockServer) {
+
+        SerenityRest
+            .given()
+            .headers(getHttpHeaders())
+            .contentType(ContentType.JSON)
+            .body("")
+            .post(mockServer.getUrl() + WA_RECONFIGURE_TASK_BY_ID)
+            .then()
+            .statusCode(204);
+
+    }
+
+    @Pact(provider = "wa_task_management_api_reconfigure_task_by_id", consumer = "wa_task_management_api")
```

Comment:
provider name does not match Provider class. Should be wa_task_management_api_reconfigure_task_by_case_id"